lib/modules/apostrophe-browser-utils/public/js/lean.js

```diff
@@ -69,17 +72,20 @@
   // delivered as well, in the second argument.
 
   apos.utils.get = function(uri, data, callback) {
+    var keys, i;
```

Comment:
It doesn't seem like these are doing anything outside of the conditional block, but also not harming anything. I'd move them in, still.

Reply:
var has no block scope, so it is considered harmful to pretend it does. Better to declare them at the top of the function because like it or not, that's the scope.

Hooray for not caring about this in 3.x.